features/feature-queries.yml

```diff
@@ -0,0 +1,16 @@
+name: Feature queries
```

Comment:
Let's make the identifier `support`. I'm not sure about the name, whatever seems most common for web developers is OK.